randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -496,34 +499,71 @@ Extra - asset_id: 3305828730
 > Door to Maintenance Tunnel; Heals? False
   * Ice Door to Maintenance Tunnel/Door to Elite Control
   * Extra - dock_index: 0
-  > Door to Elite Control Access
+  > Bottom Floor Center
       Trivial
 
 > Door to Elite Control Access; Heals? False; Spawn Point
   * Wave Door to Elite Control Access/Door to Elite Control
   * Extra - dock_index: 1
-  > Door to Maintenance Tunnel
-      Trivial
-  > Door to Ventilation Shaft
-      After Elite Control Barriers
-  > Event - Elite Control Barriers Lowered
+  > Event - Elite Pirate Defeated
+      Any of the following:
+          # https://youtu.be/mx0rQa9ZAY4
+          Charge Beam or Combat (Beginner) or Normal Damage ≥ 200
+  > Bottom Floor Center
       All of the following:
-          # Skipping Elite Pirate Standable: https://youtu.be/60i2ca6f27w
-          Scan Visor
+          After Elite Pirate Fight (Elite Control)
+          All of the following:
+              # https://youtu.be/86eWOxO83Zk
+              Movement (Intermediate) and Standable Terrain (Advanced)
 
 > Door to Ventilation Shaft; Heals? False
   * Ice Door to Ventilation Shaft/Door to Elite Control
   * Extra - dock_index: 2
-  > Door to Elite Control Access
-      After Elite Control Barriers
   > Event - Elite Control Barriers Lowered
       Scan Visor and Enabled Backwards Lower Mines
+  > Bottom Floor Center
+      After Elite Control Barriers
 
 > Event - Elite Control Barriers Lowered; Heals? False
   * Event Elite Control Barriers
-  > Door to Elite Control Access
+  > Bottom Floor Center
       Trivial
 
+> Event - Elite Pirate Defeated; Heals? False
+  * Event Elite Pirate Fight (Elite Control)
+  > Bottom Floor Center
+      Trivial
+
+> Bottom Floor Center; Heals? False
+  > Door to Maintenance Tunnel
+      Trivial
+  > Door to Elite Control Access
+      Any of the following:
+          After Elite Pirate Fight (Elite Control)
+          All of the following:
+              # https://youtu.be/lmEsr1Y1izM
+              Movement (Beginner) and Standable Terrain (Intermediate)
+  > Door to Ventilation Shaft
+      All of the following:
+          After Elite Control Barriers
+          Any of the following:
+              After Elite Pirate Fight (Elite Control)
+              All of the following:
+                  # https://youtu.be/VOesQWYfL-4
+                  Standable Terrain (Intermediate)
+  > Event - Elite Control Barriers Lowered
```

Comment:
Can you elaborate? This room has been a helpful example for the case, yes, but i don't really get which part you're talking about that "is fine for this PR". Is there any way to improve?

Reply:
You were considering N or N+1 nodes for this room. You settled on N which required 

this logic:
![image](https://user-images.githubusercontent.com/14116556/149180213-bf403aaf-1637-4a53-bddc-6eddd2892014.png)

to be written twice (duplicated).

Since there's no right or wrong way, I don't really mind either way, but the database historically usually favors not duplicating connections.

Reply:
In this case I actually added a new node already to avoid an extreme case of duplicate connections. With the current setup, no whole connection is duplicated if I didn't miss anything. I figured that if this is the case, then a new node isn't preferred.